packages/core-contracts/contracts/proxy/UnstructuredProxy.sol

```diff
@@ -0,0 +1,33 @@
+//SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import "./ForwardingProxy.sol";
+
+contract UnstructuredProxy is ForwardingProxy {
+    struct UnstructuredProxyStorage {
```

Comment:
No need to use a namespace.

Reply:
how are you going to return an address variable  in storage without using a namespace?